.github/actions/evaluations_environment_setup/action.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yamllint disable
+name: "Setup Matrix and Armory"
+
+description: "Install Armory"
+
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+
+    - name: 🌱 Setup Test Enviroment
+      shell: bash
+      run: |
+        python -m pip install --upgrade pip
+
+        pip install virtualenv
+        virtualenv .venv
+        source .venv/bin/activate
+
+    - name: ⚙️ Installing Armory
+      shell: bash
+      run: |
+        pip install --no-compile --editable '.[developer,engine,math,datasets,pytorch,datasets-builder]'
+        armory configure --use-defaults
+
+    - name: ⚙️ Installing Dependencies
+      shell: bash
+      run: |
+        sudo apt install -y protobuf-compiler
+        pip install --no-cache-dir cython
+
+        git clone https://github.com/ifzhang/ByteTrack.git
+        cd ByteTrack
+        pip3 install -r requirements.txt
+        python3 setup.py develop
+        # Requires cython for install, so will fail if run in the same pip install as cython
+        pip install --no-cache-dir cython-bbox
+
+    - name: 🐍 Load Cached Venv
+      id: cache
+      uses: actions/cache@v3
+      with:
+        path: |
+          .venv/
+        key: ${{ runner.os }}-${{ hashFiles('pyproject.toml') }}
```

Comment:
Is there some other hash that might track changes better since pyproject is not expected to change from commit to commit.  Perhaps `armory/__about__.py` if it can be expected to be present better guards cache invalidation from commit to commit.

Reply:
Appended that to the key.